### PR TITLE
Implement ObjectStorageService

### PR DESCRIPTION
**Pull request by norrland**
* Functions to call the objectstorage/ endpoints.
* Tests

### CODE REVIEW

**Comment by norrland** [comment]
I see that this doesn't want to build properly on (old) golang versions.
I will open a different pull-request regarding testing on newer versions and rebase this PR once resolved.

**Comment by norrland** [comment]
Reopen when #29 is done.

File: v2/objectstorages.go [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+// ObjectStorageInstance represents a Object Storage Instance
+type ObjectStorageInstance struct {
+	Created     string                    `json:"created"`
+	Credentials []ObjectStorageCredential `json:"credentials"`
+	DataCenter  string                    `json:"datacenter"`
+	Description string                    `json:"description,omitempty"`
+	InstanceID  string                    `json:"id"`
+}
+
+// ObjectStorageCredential represents a credential for an Object Storage Instance
```
Comment (wecc) [inline v2/objectstorages.go:21]:
```suggestion
// ObjectStorageCredential represents a credential for an Object Storage credential
```

File: v2/objectstorages.go [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+	InstanceID  string `json:"instanceid"`
+}
+
+// CreateObjectStorageCredentialParams is used when creating a new credential
+type CreateObjectStorageCredentialParams struct {
+	InstanceID  string `json:"instanceid"`
+	Description string `json:"description,omitempty"`
+}
+
+// DeleteObjectStorageCredentialParams is used when creating a new credential
```
Comment (wecc) [inline v2/objectstorages.go:48]:
```suggestion
// DeleteObjectStorageCredentialParams is used when deleting an existing credential
```

**Comment by norrland** [comment]
@wecc This should be all up to date now.

**Comment by norrland** [comment]
See #32 instead.

**Pull request closed without merging**